pkg/operators/v1alpha1/types_test.go

```diff
@@ -208,3 +209,72 @@ func TestCatalogSource_Poll(t *testing.T) {
 		require.Equal(t, tt.result, table[i].catsrc.Poll(), table[i].description)
 	}
 }
+
+func TestUpdateStrategyUnmarshal(t *testing.T) {
+	type TestStruct struct {
+		UpdateStrategy UpdateStrategy `json:"updateStrategy,omitempty"`
+	}
+	validDuration, err := time.ParseDuration("45m")
+	if err != nil {
+		return
```

Comment:
will the test succeed on this premature return?

Reply:
@perdasilva good catch, a hold up from the original PR, I've changed it to `panic` instead.